ci/cleanup.ps1

```diff
@@ -1,12 +1,27 @@
 param (
-    [string] $unreal_path = "$((Get-Item `"$($PSScriptRoot)`").parent.parent.FullName)\UnrealEngine" ## This should ultimately resolve to "C:\b\<number>\UnrealEngine".
+    [string] $unreal_path = "$((Get-Item `"$($PSScriptRoot)`").parent.parent.FullName)\UnrealEngine", ## This should ultimately resolve to "C:\b\<number>\UnrealEngine".
+    [string] $project_path = "$((Get-Item `"$($PSScriptRoot)`").parent.parent.FullName)\TestProject" ## This should ultimately resolve to "C:\b\<number>\TestProject".
```

Comment:
My thoughts on justifying the different ways in which we pass parameters:
* `setup-gdk.ps1` and `get-engine.ps1` are also used in isolation by the ExampleProject CI, which is why they need some of the provided default values and cannot assume environment variable existence.
* This `cleanup.ps1` file is also called in a pre-exit hook, which means that it may be run as part of a different shell session (possibly even one that was started before we get the chance to set environment variables). Which is why it needs some default values and cannot assume environment variable existence.
* For other scripts that do require these values as parameters, but don't define default values: I personally like to explicitly make the caller responsible for providing all required values. Environment variables bypass the normal flow of logic, similar to `GOTO` statements.

I would appreciate hearing the thoughts of a seasoned software engineer 🙂

Reply:
> seasoned software engineer

Haha you must be referring to someone else 😉 

The reason I suggested they be environment variables is they very much feel like part of the build environment (where is my engine for example). I think this is more a symptom of how its unclear (to me anyway) the dependency graph between the CI setups for the Unreal repositories (Engine, this one, Example Project, TestGym).

Reply:
Yeah, unfortunately the GDK CI is a bit of web. Are you OK with leaving the parameters as they are for now, or should I be looking into some additional refactoring?

Reply:
Yeah lets leave this for now and we can revisit it as part of a larger effort :)